docs/writing-docs/mdx.md

```diff
@@ -202,9 +203,9 @@ To help you transition to the new version, we've created a migration helper in o
 npx storybook@next automigrate mdx1to2
 ```
 
-## Documentation-only MDX
+## Extending documentation with MDX
```

Comment:
I love the narrative of extending documentation.

But, we refer to this concept as "unattached" internally (and when describing the Meta doc block). Should we make that term official, so when searching it leads here?